connectors/servicenow/src/test/java/com/vmware/connectors/servicenow/ServiceNowControllerTest.java

```diff
@@ -184,39 +163,38 @@ private ResultActions reject(String authToken, String requestFile) throws Except
 
     @Test
     void testRequestCardsUnauthorized() throws Exception {
-        mockServiceNow.expect(requestTo(any(String.class)))
+        mockBackend.expect(requestTo(any(String.class)))
                 .andRespond(withUnauthorizedRequest());
 
         requestCards(SNOW_AUTH_TOKEN, "valid/cards/card.json")
-                .andExpect(status().isBadRequest())
-                .andExpect(header().string("X-Backend-Status", "401"));
+                .expectStatus().isBadRequest()
+                .expectHeader().valueEquals("X-Backend-Status", "401");
     }
 
     @Test
     void testRequestCardsAuthHeaderMissing() throws Exception {
         requestCards(null, "valid/cards/card.json")
-                .andExpect(status().isBadRequest());
+                .expectStatus().isBadRequest();
     }
 
     @Test
     void testRequestCardsEmailNotFoundInServiceNow() throws Exception {
-        mockServiceNow.expect(requestTo("https://snow.acme.com/api/now/table/sys_user?sysparm_fields=sys_id&sysparm_limit=1&email=jbard@vmware.com"))
+        mockBackend.expect(requestTo("/api/now/table/sys_user?sysparm_fields=sys_id&sysparm_limit=1&email=jbard@vmware.com"))
                 .andExpect(header(AUTHORIZATION, "Bearer " + SNOW_AUTH_TOKEN))
                 .andExpect(method(GET))
                 .andRespond(withSuccess(fromFile("/servicenow/fake/user-not-found.json"), APPLICATION_JSON));
 
-        requestCards(SNOW_AUTH_TOKEN, "valid/cards/card.json")
-                .andExpect(status().isOk())
-                .andExpect(content().contentTypeCompatibleWith(APPLICATION_JSON))
-                .andExpect(content().string(isValidHeroCardConnectorResponse()))
-                .andExpect(
-                        content().string(
-                                JsonReplacementsBuilder
-                                        .from(fromFile("/servicenow/responses/success/cards/email-not-found.json"))
-                                        .buildForCards()
-                        )
-                );
-    }
+        String body = requestCards(SNOW_AUTH_TOKEN, "valid/cards/card.json")
+                .expectStatus().isOk()
+                .expectHeader().contentTypeCompatibleWith(APPLICATION_JSON)
+                .returnResult(String.class)
+                .getResponseBody()
+                .collect(Collectors.joining())
+                .map(JsonNormalizer::forCards)
+                .block();
+        assertThat(body, sameJSONAs(fromFile(
+                "/servicenow/responses/success/cards/email-not-found.json")).allowingAnyArrayOrdering());
```

Comment:
email-not-found, empty-email, empty-tickets, missing-email, and missing-tickets are all empty card responses.  Does this expect get simpler if you remove the parts that prep for the allowingAnyArrayOrdering?

Reply:
Yep.